byterun/main.c

```diff
@@ -21,19 +21,26 @@
 #include "caml/misc.h"
 #include "caml/mlvalues.h"
 #include "caml/sys.h"
+#include "caml/osdeps.h"
+#ifdef _WIN32
+#include <windows.h>
+#endif
 
-CAMLextern void caml_main (char **);
+CAMLextern void caml_main (charnat **);
 
 #ifdef _WIN32
-CAMLextern void caml_expand_command_line (int *, char ***);
-#endif
+CAMLextern void caml_expand_command_line (int *, wchar_t ***);
 
+int wmain(int argc, wchar_t **argv)
```

Comment:
I just want to highlight this intentional difference that libcamlrun on Windows now has a `wmain` entry point (because it's now designed for Unicode) where elsewhere, and previously, it has a `main` entry point.

This is a breaking change, but an expected one along with the changes to caml_main, etc. so C programs on Windows which invoke the runtime are going to have to change.